frontend/src/metabase/lib/settings.ts

```diff
@@ -404,4 +404,8 @@ function makeRegexTest(property: string, regex: RegExp) {
     (password.match(regex) || []).length >= (requirements[property] || 0);
 }
 
-export default new Settings(_.clone(window.MetabaseBootstrap));
+// window is not defined for static charts SSR
```

Comment:
Where are settings used in static viz?

Reply:
These settings are not used but this file gets imported since we incorrectly access settings using this global `MetabaseSettings` object which got spread across the codebase. Ideally, we should read them from the store but this is a big refactoring that is out of scope of this task

Reply:
Ahh I see.